tests/providers/google/cloud/operators/test_dataprep_system.py

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import json
+from os import environ
+
+from airflow.models import Connection
+from airflow.utils.session import create_session
+from tests.test_utils.db import clear_db_connections
+from tests.test_utils.gcp_system_helpers import CLOUD_DAG_FOLDER, GoogleSystemTest
+
+TOKEN = environ.get("DATAPREP_TOKEN", "dataprep-system-test-token")
```

Comment:
We should skip this test when this environment variable is not set, and not set a dummy value.

Reply:
What do you mean by that? This is system test, it will be skipped in CI.

Reply:
The missing variable should be detected when you try to run the test. When it is missing, a user-friendly message for the user should be displayed. We don't have tests for authorization data in environment variables yet, so you will probably need to create one.
See: 
https://github.com/apache/airflow/blob/ 7ee7d7c/TESTING.rst#adding-a-new-system-test
https://github.com/apache/airflow/blob/ 7ee7d7c/tests/conftest.py#L337-L343
https://github.com/apache/airflow/blob/ 7ee7d7c/tests/conftest.py#L248-L250